packages/config/src/env/main.js

```diff
@@ -78,11 +79,17 @@ const getGeneralEnv = async function ({
   context,
 }) {
   const gitEnv = await getGitEnv(buildDir, branch)
+
+  const DEPLOY_ID = deployId || 0
+  const BUILD_ID = buildId || 0
+
   return removeFalsy({
     SITE_ID: id,
```

Comment:
Related to https://github.com/netlify/build/pull/3931/files#r767883705, we might consider also adding a default value (maybe `0`?) to `SITE_ID` as well.